lib/renderer/api/desktop-capturer.js

```diff
@@ -1,46 +1,43 @@
-const ipcRenderer = require('electron').ipcRenderer
-const nativeImage = require('electron').nativeImage
+const {ipcRenderer, nativeImage} = require('electron')
 
-var nextId = 0
-var includes = [].includes
+const includes = [].includes
+let currentId = 0
 
-var getNextId = function () {
-  return ++nextId
+const incrementId = () => {
+  currentId += 1
+  return currentId
 }
 
 // |options.type| can not be empty and has to include 'window' or 'screen'.
```

Comment:
This comment doesn't match that the function actually does =/

Reply:
lol yikes i didn't even notice that, changing now!